style.css

```diff
@@ -382,21 +427,355 @@ a.btn2 {
   padding-left: 10px;
 }
 
-.hi {
-  background: #1c1a19;
-  padding-top: 20px;
-  margin-top: -22px;
-  padding-bottom: 30px;
-  text-align: center;
-  padding-right: 50px;
+.foot {
+  justify-content: center;
+  padding: 12px 24px;
 }
 
-.di {
-  text-align: center;
-  font-family: "InaiMathi", sans-serif;
+.aa {
+  flex-wrap: wrap;
+  display: flex;
+  align-items: center;
+}
+
+.aa div {
+  justify-items: center;
+  flex-grow: 1;
+  flex-basis: 325px;
+}
+
+.ab {
+  justify-content: center;
+  font-family: 'Inter', sans-serif;
   font-style: normal;
   font-weight: 700;
-  font-size: 32px;
+  font-size: 24px;
   line-height: 44px;
+  text-align: center;
+  color: #172b4d;
```

Comment:
- [ ] [REQUIRED] Please change the `color: #172b4d;` to correct `color: #262626;` 

![image](https://user-images.githubusercontent.com/63915024/186001979-e8aa86e2-4950-4403-8719-f9e8db0306f9.png)

Reply:
Done